src/lib/umd.index.ts

```diff
@@ -62,7 +61,6 @@ const SynapseComponents = {
   UserCard,
   StatisticsPlot,
   HasAccess,
-  HasAccessV2,
```

Comment:
Replace HasAccess with v2. Use the same exported name, so no need to update SWC.